### PR TITLE
T1 jsintegration

### PR DESCRIPTION
**Pull request by Abhijeett22**
Added addition, subtract, multiply, divide functionality

### CODE REVIEW

File: calculator.js [hunk 1]
```diff
@@ -2,87 +2,80 @@
...
                 // Current calcContent is 0 and its assigned to calcDisplay 
-                calcDisplay.textContent = calcContent;
+                 calcDisplay.value = calcContent;
             } else {
-                 // on data action add number in display area
-                calcDisplay.textContent = displayClickedValue + calcContent;
+                // on data action add number in display area
+                calcDisplay.value = displayClickedValue + calcContent;
+                // sum += parseFloat(displayClickedValue);
+                if (operand === 'subtract') {
```
Comment (vijay-gawade18) [inline calculator.js:35]: You have assigned boolean value to the operand and here, you are trying to reassign the string value. Why? Is it recommended practise?

File: calculator.js [hunk 2]
```diff
@@ -2,87 +2,80 @@
...
+                 if (operand === 'addition') {
+                    sum += parseFloat(calcContent);
+                 } 
+                 
+                 if (operand === 'multiply') {
+                    sum *= parseFloat(calcContent);
+                 }
+                 
+                 if (operand === 'divide') {
+                    sum *= parseFloat(calcContent);
```
Comment (vijay-gawade18) [inline calculator.js:47]: Change the operator from * to /

File: calculator.js [hunk 3]
```diff
@@ -2,87 +2,80 @@
...
+               
             }
-        }
+        } 
+        
         // In case of data-action property as operator 
         if(calcAction === 'addition' || calcAction === 'subtract' || calcAction === 'multiply' || calcAction === 'divide') {
             console.log('this is operator');
+            calcDisplay.value = displayClickedValue + calcContent;
+             if (calcAction === calcAction) {
```
Comment (vijay-gawade18) [inline calculator.js:57]: What are you trying to do here in if condition. '**calcAction === calcAction**' what does it mean?

File: calculator.js [hunk 4]
```diff
@@ -2,87 +2,80 @@
 const calculator = document.querySelector('.calculator-main');
 const calcKeys = calculator.querySelector('.calc-keys');
 const calcDisplay = document.querySelector('.calc-display');
+let sum = 0; 
+let stroevalue;
```
Comment (vijay-gawade18) [inline calculator.js:6]: change 'stroevalue' to 'storeValue'.

**Comment by Abhijeett22** [comment]
changes fixed

File: calculator.js (hunk 1 above)
Comment (Abhijeett22) [reply calculator.js:35]: removed boolean value and assigned blank value

File: calculator.js (hunk 2 above)
Comment (Abhijeett22) [reply calculator.js]: changed * to /

File: calculator.js (hunk 3 above)
Comment (Abhijeett22) [reply calculator.js]: remove compassion condition

**Pull request merged**